application/config.json.template

```diff
@@ -1,5 +1,6 @@
 {
     "token": "<put_your_token_here>",
+    "gistApiKey": "<your_gist_personal_access_token>",
```

Comment:
Since you are changing the config, please add to your PR description that a config change is needed (and what to change).

That way it is easier for someone doing the releases (like me) to know what to do. And to not forget.